Buckify op_add for FusionG3 and add cxx tests.

hsharma35 opened the pull request:
Differential Revision: D66532566

hsharma35 commented:
`@pytorchbot label "topic: not user facing"`

zonglinpeng reviewed (approved):
LGTM, thanks for adding the actual unit test per op!

can you please add cmake files and instructions to run the test under [Cadence readme](https://github.com/pytorch/executorch/tree/main/examples#cadence-hifi4-dsp) as a new sub-topic, e.g Cadence G3

zonglinpeng commented on backends/cadence/fusion_g3/operators/tests/test_op_add.cpp:
```diff
@@ -0,0 +1,84 @@
+  Tensor out = tf.zeros(sizes);
+
+  // Add two 2x2 tensors.
+  add_out(tf.make(sizes, {1, 2, 3, 4}), tf.make(sizes, {2, 2, 2, 2}), 1, out);
```
is there a way to parameterize the test inputs for multiple shapes>

hsharma35 committed:
Buckify op_add for FusionG3 and add cxx tests. (#7102)

Summary:

Minimal buckification for `op_add` with some tests to reproduce failures.

Reviewed By: zonglinpeng

Differential Revision: D66532566
